meshnesh/android-level-up
# #156037248 UI/UX revamp

# What does this PR do?

Revamp the Applications UI/UX

# Description of Task to be completed?

* Switch the application from using a ListView to GridView Layout.
* Implement  SwipeToRefresh.
* Implement ProgressDialog.

## How should this be manually tested?

* Clone the repo.
* Sync/build app.
* Launch app with the device emulator.
* Swipe down on the app to refresh the application and see the dialog.

## What are the relevant pivotal tracker stories?

[#156037248](https://www.pivotaltracker.com/story/show/156037248)

### Screenshots (if appropriate)?

![ui-revamp](https://user-images.githubusercontent.com/15923109/42767718-6a1542b8-8926-11e8-9661-c5d3a76085d2.gif)
